custom_components/reflex_ag_grid/ag_grid.py

```diff
@@ -353,24 +353,24 @@ def api(self) -> AgGridAPI:
 
     def getSelectedRows(self, callback: rx.EventHandler) -> rx.event.EventSpec:
         return rx.call_script(
-            f"refs['{self.get_ref()}']?.current?.api.getSelectedRows()",
+            f"refs['{self.api}.getSelectedRows()",
```

Comment:
still doesn't seem quite right, did you try it in the browser?

i think just `return self.api.getSelectedRows()` would be sufficient as below

Reply:
nevermind, i was looking at old code